spring-cloud-dataflow-server-core/src/main/resources/org/springframework/cloud/dataflow/server/db/migration/h2/V1__INITIAL_SETUP.sql

```diff
@@ -26,7 +26,7 @@ create table app_registration (
 create table audit_records (
   id bigint not null,
   audit_action bigint,
-  audit_data varchar(2147483647),
+  audit_data varchar(4000),
```

Comment:
In general, H2 has gotten stricter about what they allow and the syntax/parsing. The huge VARCHAR is no longer a thing. This should be big enough for testing purposes.